pkg/valueset/postgresql.go

```diff
@@ -0,0 +1,33 @@
+package pkg
```

Comment:
Move into pgsql package

pkg/valueset/pgsql

Then we'll write some interfaces to valueset pkg